### PR TITLE
Feature/hw 4

### PR DESCRIPTION
**Pull request by LevapVeeskela**
hw_4

### CODE REVIEW

File: ItAcademyHomeWorks/Common/Helpers/HwFourSecondTaskHelper.cs [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+            HomeWorkFour.TextCommandMoveUpInfo => HomeWorkFour.TextMoveUpInfo,
+            HomeWorkFour.TextAlternativeCommandMoveUpInfo => HomeWorkFour.TextMoveUpInfo,
+            HomeWorkFour.TextCommandMoveDownInfo => HomeWorkFour.TextMoveDownInfo,
+            HomeWorkFour.TextAlternativeCommandMoveDownInfo => HomeWorkFour.TextMoveDownInfo,
+            HomeWorkFour.TextCommandMoveRightInfo => HomeWorkFour.TextMoveRightInfo,
+            HomeWorkFour.TextAlternativeCommandMoveRightInfo => HomeWorkFour.TextMoveDownInfo,
+            HomeWorkFour.TextCommandMoveLeftInfo => HomeWorkFour.TextMoveLeftInfo,
+            HomeWorkFour.TextAlternativeCommandMoveLeftInfo => HomeWorkFour.TextMoveLeftInfo,
+            _ => HomeWorkFour.TextWrongCommandInfo
+        };
```
Comment (VadzimPapko) [inline ItAcademyHomeWorks/Common/Helpers/HwFourSecondTaskHelper.cs]: new c# 8 feature - well done!

File: ItAcademyHomeWorks/Common/Enums/HwFourCommandsEnum.cs [hunk 2]
```diff
@@ -0,0 +1,10 @@
+﻿namespace Common.Enums
+{
+    public enum HwFourCommandsEnum
+    {
+        Down = 2,
+        Left = 4,
+        Right = 6,
+        Up = 8
```
Comment (VadzimPapko) [inline ItAcademyHomeWorks/Common/Enums/HwFourCommandsEnum.cs]: why have you assigned numbers even 2?

Comment (LevapVeeskela) [reply ItAcademyHomeWorks/Common/Enums/HwFourCommandsEnum.cs]: this enum not 2^, this user enum, the layout of the arrows on the numbers on my keyboard...

File: ItAcademyHomeWorks/Common/Helpers/HwFourSecondTaskHelper.cs (hunk 1 above)
Comment (LevapVeeskela) [reply ItAcademyHomeWorks/Common/Helpers/HwFourSecondTaskHelper.cs]: thanks) I try to improve my skills

**Pull request merged**